### PR TITLE
Updated README

### PR DESCRIPTION
**Pull request by davekaj**
Complete re-write. It was very out of date.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,90 +1,149 @@
 # Graph Protocol Solidity Smart Contracts
-
+![License: MIT](https://img.shields.io/badge/license-MIT-green)
```
Comment (abarmat) [inline README.md:2]: This should be GPL now

File: README.md [hunk 2]
```diff
@@ -1,90 +1,149 @@
...
 ![Version Badge](https://img.shields.io/badge/version-1.6.0-lightgrey.svg)
 ![CI Status](https://github.com/graphprotocol/contracts/actions/workflows/npmtest.yml/badge.svg)
 [![codecov](https://codecov.io/gh/graphprotocol/contracts/branch/dev/graph/badge.svg?token=S8JWGR9SBN)](https://codecov.io/gh/graphprotocol/contracts)
+# The Graph Protocol Contracts
+[The Graph](https://thegraph.com/) is an indexing protocol for querying networks like Ethereum,
```
Comment (abarmat) [inline README.md:7]: There are some line breaks here that makes it look odd when looking the preview markdown. Maybe an external editor added those.

File: README.md (hunk 1 above)
Comment (davekaj) [reply README.md:2]: updated

File: README.md (hunk 2 above)
Comment (davekaj) [reply README.md]: updated

**Review by abarmat — APPROVED** [review]

**Pull request merged**